### PR TITLE
Repo Descriptions for Go

### PR DESCRIPTION
**Pull request by JesterOrNot**

### CODE REVIEW

File: src/docs/go-in-gitpod.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ Before we start, here are some already gitpodified repositories to give you an i
...
+| Repository                                                              |                                                                 Description                         |               Try It |
+| ----------------------------------------------------------------- | ------------------------------|-------------------------------------------------------------------------------------------------: |
+| [go-swagger/go-swagger](https://github.com/go-swagger/go-swagger) |Swagger 2.0 implementation for go| [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/go-swagger/go-swagger) |
+| [gitpod-io/go-gin-app](https://github.com/gitpod-io/go-gin-app)  | gin example running in Gitpod |  [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/go-gin-app) |
+| [gosh-terminal/gosh](https://github.com/gosh-terminal/gosh)   | Do everything from the terminal  |    [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gosh-terminal/gosh) |
```
Comment (jankeromnes) [inline src/docs/go-in-gitpod.md]: This is not really a good description of what this example repo is.

File: src/docs/go-in-gitpod.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ Before we start, here are some already gitpodified repositories to give you an i
...
-| [gitpod-io/self-hosted](https://github.com/gitpod-io/self-hosted) | [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/self-hosted) |
+| Repository                                                              |                                                                 Description                         |               Try It |
+| ----------------------------------------------------------------- | ------------------------------|-------------------------------------------------------------------------------------------------: |
+| [go-swagger/go-swagger](https://github.com/go-swagger/go-swagger) |Swagger 2.0 implementation for go| [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/go-swagger/go-swagger) |
+| [gitpod-io/go-gin-app](https://github.com/gitpod-io/go-gin-app)  | gin example running in Gitpod |  [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/go-gin-app) |
```
Comment (jankeromnes) [inline src/docs/go-in-gitpod.md]: Starting with a capital letter is nicer :)

```suggestion
| [gitpod-io/go-gin-app](https://github.com/gitpod-io/go-gin-app)  | Gin example running in Gitpod |  [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/go-gin-app) |
```

File: src/docs/go-in-gitpod.md [hunk 3]
```diff
@@ -8,12 +8,12 @@ Before we start, here are some already gitpodified repositories to give you an i
...
-| [gosh-terminal/gosh](https://github.com/gosh-terminal/gosh)       |    [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gosh-terminal/gosh) |
-| [gitpod-io/self-hosted](https://github.com/gitpod-io/self-hosted) | [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/self-hosted) |
+| Repository                                                              |                                                                 Description                         |               Try It |
+| ----------------------------------------------------------------- | ------------------------------|-------------------------------------------------------------------------------------------------: |
+| [go-swagger/go-swagger](https://github.com/go-swagger/go-swagger) |Swagger 2.0 implementation for go| [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/go-swagger/go-swagger) |
```
Comment (jankeromnes) [inline src/docs/go-in-gitpod.md]: Spaces and capitalized Go (slight deviation from official repo description):

```suggestion
| [go-swagger/go-swagger](https://github.com/go-swagger/go-swagger) | Swagger 2.0 implementation for Go | [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/go-swagger/go-swagger) |
```

File: src/docs/go-in-gitpod.md (hunk 1 above)
Comment (JesterOrNot) [reply src/docs/go-in-gitpod.md]: Which one?

Comment (JesterOrNot) [reply src/docs/go-in-gitpod.md]: Never mind I think I know

**Comment by JesterOrNot** [comment]
Any better?

File: src/docs/go-in-gitpod.md (hunk 1 above)
Comment (jankeromnes) [reply src/docs/go-in-gitpod.md]: Look at the exact line this comment was posted on :) "Do everything from the terminal" does not tell me anything about what this project is about

Comment (jankeromnes) [reply src/docs/go-in-gitpod.md]: FYI the previous go-swagger description was better

**Pull request merged**